libs/libqmi/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libqmi
+PKG_VERSION:=1.22.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.freedesktop.org/software/libqmi
+PKG_HASH=9056e8ab6e26af014d5bd1b29cc439f6bdb46994e0bd5e80f28947ae6f38a414
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+
+PKG_FIXUP:=autoreconf
```

Comment:
Is it realy needed? I tested without autoreconf and all is OK.

Reply:
autoreconf is only needed if bulding from a git snapshot. If we're building from a source tarball, there is absolutely no need for autoreconf